packages/salesforcedx-vscode-core/test/jest/commands/utils/parameterGatherers.test.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as vscode from 'vscode';
+import {
+  CANCEL,
+  CONTINUE,
+  SelectFileName
+} from '../../../../src/commands/util/parameterGatherers';
+import { nls } from '../../../../src/messages';
+
```

Comment:
These are pretty weird to test b/c they depend on some vscode internal functionality.  Not my fav to have to mock implementation of showInputBox in each test, but the best flow I could come up with for this validation.